cli/src/services/project-fetcher.ts

```diff
@@ -32,13 +32,13 @@ export default class ProjectFetcher {
   }
 
   private async graphql(config: Config, params: object) {
-    const query = `query ProjectDetails($projectId: ID! $versionId: ID) {
+    const query = `query ProjectDetails($project_id: ID! $versionId: ID) {
```

Comment:
@simonprev , i commited the fix, but i can remove it if you planned to push it in another PR.

Reply:
Done!